lib/clif-backend/src/lib.rs

```diff
@@ -43,16 +43,22 @@ fn get_isa(config: Option<&CompilerConfig>) -> Box<dyn isa::TargetIsa> {
         builder.set("opt_level", "speed_and_size").unwrap();
         builder.set("enable_jump_tables", "false").unwrap();
 
-        if cfg!(test) || cfg!(debug_assertions) {
-            builder.set("enable_verifier", "true").unwrap();
-        } else {
-            builder.set("enable_verifier", "false").unwrap();
-        }
+        let enable_verifier: bool;
 
         if let Some(config) = config {
             if config.nan_canonicalization {
                 builder.set("enable_nan_canonicalization", "true").unwrap();
             }
+            enable_verifier = !config.disable_debug_mode_verification;
+        } else {
+            // Set defaults if no config found.
+            enable_verifier = true;
```

Comment:
How about `enable_verifier = cfg!(test) || cfg!(debug_assertions)`?

Reply:
That would require duplicating that boolean expression because it needs to be checked on the other case too!  I considered it but decided against it